app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -721,42 +748,48 @@ class InputInteractionViewTestActivityTest {
 
   @Test
   fun testRatioInputInteractionView_withNoInputText_hasCorrectPendingAnswerType() {
-    val activityScenario = ActivityScenario.launch(
+    launch(
       InputInteractionViewTestActivity::class.java
     )
-    activityScenario.use {
-      activityScenario.onActivity { activity ->
+    .use {
+      testCoroutineDispatchers.advanceUntilIdle()
+      it.onActivity { activity ->
```

Comment:
@anandwana001 I tried fixing test cases by `.runcurrenr()` as well as `.advancedUntilIdle()` but none of them worked